all-repos.yaml

```diff
@@ -140,3 +140,4 @@
 - https://github.com/hukkinj1/mdformat
 - https://gitlab.com/daverona/pre-commit/php
 - https://github.com/anderseknert/pre-commit-opa
+- https://github.com/radix-ai/auto-smart-commit
```

Comment:
I'd suggest removing the suggestion of `default_stages` from your example configuration, maybe suggest it but don't require it (it doesn't always do what someone would want)

Reply:
Removed that line from the README.